build/external_extension/WikibaseLocalMedia.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+## WikibaseLocalMedia
+## NOTE: WikibaseLocalMedia does currently not work in a client only setup.
+UPDATE_SUBMODULE=0 bash build/clone_repo.sh \
+    "$WIKIBASELOCALMEDIA_COMMIT_HASH" \
+    "git_cache/WikibaseLocalMedia.git" \
+    WIKIBASELOCALMEDIA \
+    "$BUILT_EXTENSIONS_PATH/WikibaseLocalMedia" \
+    master && bash build/clean_repo.sh "$BUILT_EXTENSIONS_PATH/WikibaseLocalMedia"
```

Comment:
(Doesn’t have to be changed now, since it was this way in the old code, but) I find it pretty confusing how this has one command split across multiple lines and then at the end a second command on the same line.

Reply:
done